splice_machine/src/main/java/com/splicemachine/backup/BackupSystemProcedures.java

```diff
@@ -269,7 +270,7 @@ public static void SYSCS_DELETE_OLD_BACKUPS(int backupWindow, ResultSet[] result
         }
     }
 
-    public static void CANCEL_BACKUP(ResultSet[] resultSets) throws StandardException, SQLException {
+    public static void SYSCS_CANCEL_BACKUP(ResultSet[] resultSets) throws StandardException, SQLException {
```

Comment:
Making this change will require additional steps when performing upgrades to this version: not severe (just need to reload stored procedures and change application code which uses it), but painful enough I have to ask if it's worth it to make the change.